Blog/What are Ethereum killers

```diff
@@ -0,0 +1,112 @@
+# What are Ethereum killers?
+
```

Comment:
Line 3 is not visible as the image .Means  to get ,what I am saying u have to just click on the preview available on the right side .Inside the Three dots . u will get to know what I meant .
Feel free to asl if u need any help .
Thanks .